types/react-slick/index.d.ts

```diff
@@ -9,9 +11,13 @@ import * as React from "react";
 type ComponentConstructor<TProps> = React.ComponentClass<TProps> | React.StatelessComponent<TProps>;
 
 export interface CustomArrowProps {
+  key?: string | number;
```

Comment:
`key` already exists in React component.

Reply:
Ah, right. Sorry … Will fix it.

Reply:
Fixed it. Please review.